index.js

```diff
@@ -216,25 +216,25 @@ async function main (argv) {
     process.exit(1)
   }
 
-  process.stdout.write('Ok.')
+  process.stdout.write('Ok.\n')
 
   //
   // Initialize the current directory as a socket app.
   //
   try {
-    process.stdout.write('\nCreating socket files...')
+    process.stdout.write('Creating socket files...')
     await exec('npm run init-project')
   } catch (err) {
     process.stderr.write(`\nUnable to create socket files: ${err.stack ?? err.message}\n`)
   }
-  process.stdout.write('Ok.')
+  process.stdout.write('Ok.\n')
 
   //
   //  Initialize tsconfig.json when react_ts
   //
   if (templateName === 'react-ts') {
     try {
-      process.stdout.write('\nCreating tsconfig...')
+      process.stdout.write('Creating tsconfig...')
```

Comment:
Just checking, this one doesn't need a trailing `\n`?

Reply:
It doesn't need.

```
Creating tsconfig...Ok.
```